### ISSUE TITLE
Add pop-up at cash out for minimum $50.

### ISSUE THREAD

**Issue by MaxTattenbach** [opencurrents/opencurrents#1027]
@dannypernik  let's add a pop-up when user clicks cash out in which it displays the following. 

During our beta phase, you will need at least $50 to cash out.  If you have more than $50 and want to cash out please [click here] and let us know.    To exchange more Currents for USD visit the market-place.

Let's add button in pop-up which says,  Got it and goes to home page. 

Link should take us to twak box.

**Comment by MaxTattenbach** [comment]
Note that existing pop-up always says "No Available Dollars"  I clicked cash out with $37 in my account and this pop-up displayed. 

![image](https://user-images.githubusercontent.com/26234440/35249312-1f664992-ff97-11e7-9264-9cfdb35d0424.png)

**Comment by nickolashe** [comment]
@dannypernik since this is on /profile you should be able to use the pending usd context var i think?

**Comment by dannypernik** [comment]
@nickolashe @MaxTattenbach I think we should say "You need to have more than $40 in Available Dollars to cash out" instead of $50. A likely scenario is that a user redeems all $40 from master offer, then redeems an offer worth less than $10. I think we should let him/her cash out in this case.

Nicko I tried to test the `{% if balance_available_usd|str_to_num > 40 %}` by changing a TransactionAction to 'redeemed'. This reduced my pending Currents but did not increase Dollars available. Is there something else that needs to happen in order to increase Available dollars?

<img width="932" alt="screen shot 2018-01-23 at 1 55 37 pm" src="https://user-images.githubusercontent.com/3347095/35297605-2013339c-0045-11e8-9f1c-9e405eff0a98.png">

**Comment by nickolashe** [comment]
@dannypernik you'd need a ledger record for the user's entity in currency "usd". for reference, here is the implementation of get_balance_available_usd:
https://github.com/opencurrents/opencurrents/blob/develop/openCurrents/interfaces/ocuser.py#L180

**Comment by dannypernik** [comment]
@nickolashe I get this error when trying to add a ledger in admin panel. Seems like I should clear my DB. Also it would be helpful to get a rundown of DB to know which data tables require matching queries from another table. I assume I'm creating errors when trying to add or delete data during testing.

![image](https://user-images.githubusercontent.com/3347095/35428423-908f4c82-0234-11e8-959f-a0fbe796dd58.png)

**Comment by nickolashe** [comment]
yes, something like this is usually a remnant of the old model. its best to clear and use the fixtures script for testing.
